examples/sampling/graphbolt/node_classification.py

```diff
@@ -0,0 +1,331 @@
+
+            # The input features from the source nodes in the first layer's
+            # computation graph.
+            # x = blocks[0].srcdata["feat"]
+            x = data.node_features["feat"].float()
+
+            # print(x)
+
+            # The ground truth labels from the destination nodes
+            # in the last layer's computation graph.
+            # y = blocks[-1].dstdata["label"]
+            y = data.labels
+
+            # The predicted labels.
+            y_hat = model(data.to_dgl_blocks(), x)
+
+            # Compute loss.
+            loss = F.cross_entropy(y_hat, y)
+
+            optimizer.zero_grad()
+            loss.backward()
+            optimizer.step()
+
+            total_loss += loss.item()
+
+        # Evaluate the model.
+        print("Validating...")
+        acc = evaluate(args, model, graph, features, valid_set, num_classes)
+        print(
+            f"Epoch {epoch:05d} | Loss {total_loss / (step + 1):.4f} | "
+            f"Accuracy {acc.item():.4f} "
+        )
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description="A script trains and tests a GraphSAGE model "
+        "for node classification using GraphBolt dataloader."
+    )
+    parser.add_argument(
+        "--epochs", type=int, default=10, help="Number of training epochs."
+    )
+    parser.add_argument(
+        "--lr",
+        type=float,
+        default=0.0005,
+        help="Learning rate for optimization.",
+    )
+    parser.add_argument(
+        "--batch-size", type=int, default=512, help="Batch size for training."
+    )
+    parser.add_argument(
+        "--num-workers",
+        type=int,
+        default=4,
+        help="Number of workers for data loading.",
+    )
+    parser.add_argument(
+        "--fanout",
+        type=str,
+        default="15,10,5",
+        help="Fan-out of neighbor sampling. It is IMPORTANT to keep len(fanout)"
+        " identical with the number of layers in your model. Default: 15,10,5",
+    )
+    parser.add_argument(
+        "--device",
+        default="cpu",
+        choices=["cpu", "cuda"],
+        help="Train device: 'cpu' for CPU, 'cuda' for GPU.",
+    )
+    return parser.parse_args()
+
+
+def main(args):
+    if not torch.cuda.is_available():
+        args.device = "cpu"
+    print(f"Training in {args.device} mode.")
+
+    # Load and preprocess dataset.
+    dataset = gb.OnDiskDataset(
+        "/home/ubuntu/example_ogbn_products"
+    ).load()  # Please edit it according to the path of your own dataset.
+
+    graph = dataset.graph
+    features = dataset.feature
+    train_set = dataset.tasks[0].train_set
+    valid_set = dataset.tasks[0].validation_set
+    args.fanout = list(map(int, args.fanout.split(",")))
+
+    num_classes = dataset.tasks[0].metadata["num_classes"]
+
+    in_size = features.read("node", None, "feat").shape[
+        1
+    ]  # Size of feature of a single node.
+    hidden_size = 128  # Customized by you.
```

Comment:
Avoid using personal pronouns (I, you, he) in the code.
```suggestion
    hidden_size = 128
```